private_config.php

```diff
@@ -254,30 +253,30 @@
                                 if($index == $faq['help_tab']) {
                                     if ($faq['help_category'] == $category_id && $faq['help_show_y'] != "0") {
                                         if ($category_count == 0) {
-                                            echo '<div class="faqHeader">' . $help_category[$faq['help_category']] . '</div>';
+                                            echo '<div class="faqHeader">' . htmlentities($help_category[$faq['help_category']],ENT_QUOTES) . '</div>';
                                         }
                                         $category_count++;
                                         $collapse_id = "category_" . $category_id . "_question_" . $category_count."_tab_".$index;
 
                                         echo '<div class="panel panel-default">
                                     <div class="panel-heading">
                                         <h4 class="panel-title">
-                                            <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion" href="#' . $collapse_id . '">' . $faq['help_question'] . '</a>
+                                            <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion" href="#' . htmlentities($collapse_id,ENT_QUOTES) . '">' . filter_tags($faq['help_question']) . '</a>
                                         </h4>
                                     </div>
-                                    <div id="' . $collapse_id . '" class="panel-collapse collapse">
+                                    <div id="' . htmlentities($collapse_id,ENT_QUOTES) . '" class="panel-collapse collapse">
                                         <div class="panel-body">
-                                            <div>' . $faq['help_answer'] . '</div>';
+                                            <div>' . filter_tags($faq['help_answer']) . '</div>';
 
 
-                                        echo printFile($module, $faq['help_image'], 'img');
-                                        echo printFile($module, $faq['help_document'], 'doc');
-                                        echo printFile($module, $faq['help_document2'], 'doc');
+                                        echo filter_tags(\Vanderbilt\FaqMenuExternalModule\printFile($module, $faq['help_image'], 'img'));
+                                        echo filter_tags(\Vanderbilt\FaqMenuExternalModule\printFile($module, $faq['help_document'], 'doc'));
+                                        echo filter_tags(\Vanderbilt\FaqMenuExternalModule\printFile($module, $faq['help_document2'], 'doc'));
```

Comment:
I believe the 'Vanderbilt\FaqMenuExternalModule' prefix here and other places isn't needed since you're namespacing everything. But it doesn't hurt anything to have it there.